qa/tasks/mgr/dashboard/test_pool.py

```diff
@@ -44,8 +44,22 @@ class PoolTest(DashboardTestCase):
     }))
 
     @contextmanager
-    def __create_pool(self, name, data=None):
-        pool_data = data or {
+    def __yield_pool(self, name=None, data=None, deletion_name=None):
+        """
+        Use either just a name or whole description of a pool to create one.
+        This also validates the correct creation and deletion after the pool was used.
+
+        :param name: Name of the pool
+        :param data: Describes the pool in full length
+        :param deletion_name: Only needed if the pool was renamed
+        :return:
+        """
+        data = self._create_pool(name, data)
```

Comment:
Please, don't reuse input arguments for internal variables.

Reply:
It's not a real reuse as it's still the same type of content - it's the description data to create a pool - it will automatically declared if you only specified a pool name.